README.md

```diff
@@ -55,27 +55,25 @@ npm install -g @nanoapi.io/napi
 
 ```bash
 napi init
-napi ui
+napi split configure
+napi split
 ```
 
 This will initialize a new NanoAPI project and open the UI for inspecting and refactoring your codebase.
 
-## CLI Usage
+**Note:** `napi` works off a simple system of annotation. These annotation allow `napi` to be integrated into any existing projects. Annotating a program is needed before using the split functionalities see [Refactoring with Annotations](#refactoring-with-annotations).
```

Comment:
I think better to add it here to make sure people read it right after running the command. Otherwise they might be confuse by the fact that an unannotated program cannot be splitted (assuming they did not read the annotation part of the readme which is further down.